test/test_adistream.py

```diff
@@ -0,0 +1,190 @@
+import argparse
+
+import pytest
+from adi.tools.adistream import run_adi_stream
```

Comment:
You'll need an import check to see if adi tools was imported. Otherwise this will always break tests when the dependencies don't exist

Reply:
Yes, ModuleNotFoundError is occurring when importing Genalyzer. Shall I skip the tests if an import error occurs here?

Reply:
May I know if any more changes need to be made, @tfcollins?